demo-notebooks/additional-demos/hf_interactive.ipynb

```diff
@@ -100,7 +98,7 @@
     "                                       max_cpus=8, \n",
     "                                       min_memory=16, \n",
     "                                       max_memory=16, \n",
-    "                                       image=\"quay.io/project-codeflare/ray:latest-py39-cu118\",\n",
+    "                                       image=\"quay.io/project-codeflare/ray:2.20.0-py39-cu118\",\n",
```

Comment:
```suggestion
    "                                       image=\"quay.io/project-codeflare/ray:latest-py39-cu118\",\n",
```
We can put this back for now. I plan to update all of the demo notebooks with new images once the ray version PR #530 is in

Reply:
Perfect, I've reverted it in all places. Thanks @Bobbins228